db/migrations/20220414100500.json

```diff
@@ -0,0 +1,4 @@
+{
+  "up": "ALTER TABLE people ADD approved BOOLEAN AFTER login_enabled",
+  "down": "ALTER TABLE people DROP COLUMN approved"
+}
```

Comment:
This has been moved to @klaudsol/commons
https://github.com/klaudsol/klaudsol-commons/blob/main/dist/db/migrations/20220414100500.json